Documentation/Testing/UnitTesting/Running.rst

```diff
@@ -0,0 +1,129 @@
+ 6.x.x              v10, v11         7.2, 7.3, 7.4, 8.0, 8.1, 8.2, 8.3   ^8, ^9
+================== ================ =================================== ==========
+
+Testing framework <= 6.x no longer maintained.
+
+..  _testing-unit-run-configure:
+
+Provide configuration files for unit tests
+==========================================
+
+The TYPO3 testing framework comes with a predefined unit test configuration and
+a bootstrapping file. You should copy these files into your project so you
+can adjust them as needed:
+
+Copy the files `vendor/typo3/testing-framework/Resources/Core/Build/UnitTests.xml
+<https://github.com/TYPO3/testing-framework/blob/main/Resources/Core/Build/UnitTests.xml>`__
+and `vendor/typo3/testing-framework/Resources/Core/Build/UnitTestsBootstrap.php
+<https://github.com/TYPO3/testing-framework/blob/main/Resources/Core/Build/UnitTestsBootstrap.php>`__
+
+Open file :file:`UnitTests.xml` and adjust the paths to the path or paths where
+the unit tests are stored. By convention many extensions store them in the
+directory :path:`Tests/Unit` and subdirectories thereof:
+
+..  code-block:: diff
+    :caption: UnitTests.xml for extension testing
+
+    <testsuites>
+        <testsuite name="Unit tests">
+    -        <directory>../../../../../../typo3/sysext/*/Tests/Unit/</directory>
+    +        <directory>../../Tests/Unit/</directory>
+        </testsuite>
+    </testsuites>
+
+If you are testing in a project you will probably have a directory from where
+local extensions like site packages and client-specific extensions are installed
+you should then also include them:
+
+..  code-block:: diff
+    :caption: UnitTests.xml for project testing
+
+    <testsuites>
+        <testsuite name="Unit tests">
+    -        <directory>../../../../../../typo3/sysext/*/Tests/Unit/</directory>
+    +        <directory>../../Tests/Unit/</directory>
+    +        <directory>../../packages/*/Tests/Unit/</directory>
+        </testsuite>
+    </testsuites>
+
+Run the unit tests on your system or with DDEV
+==============================================
+
+If you have the required PHP version installed on your system, you can run the
+unit tests directly:
+
+..  code-block:: bash
+
+    php vendor/bin/phpunit -c Build/phpunit/UnitTests.xml
+
+Or you can run them on ddev:
+
+..  code-block:: bash
+
+    ddev exec php vendor/bin/phpunit -c Build/phpunit/UnitTests.xml
+
+If you are using DDEV, you can also create a `custom command <https://ddev.readthedocs.io/en/stable/users/extend/custom-commands/#custom-commands>`__
+so you can run these tests again and again.
+
+If you want to only run a specific test case (test class) or test (method in a
+test class) you can use the filter option:
+
+..  code-block:: bash
+
+    php vendor/bin/phpunit -c Build/phpunit/UnitTests.xml --filter "MyTest"
+
+You can of course define a
+`Composer script <https://getcomposer.org/doc/articles/scripts.md>` as well.
```

Comment:
```suggestion
`Composer script <https://getcomposer.org/doc/articles/scripts.md>` as well, so that this command can be executed easily on the host, within a DDEV container and also in GitHub Actions or Gitlab CI.
```